### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by taion**
Main change here is to ignore coverage, but it seems like we should ignore nested `lib/` and `es/` as well?

### CODE REVIEW

File: commands/new.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ async function setupNpm(dest, a) {
       version: '1.0.0',
       main: a.babel ? 'lib/index.js' : 'index.js',
       ...(a.babel && {
-        modules: 'es/index.js',
+        module: 'es/index.js',
```
Comment (jquense) [inline commands/new.js]: is it not `modules`?

Comment (taion) [reply commands/new.js]: https://webpack.js.org/configuration/resolve/#resolve-mainfields

[the remark this answers is not in the document]
Comment (jquense) [reply commands/new.js]: i could have sworn it was plural but that's so webpack. siigh

[the remark this answers is not in the document]
Comment (taion) [reply commands/new.js]: rollup uses `module` now too

[the remark this answers is not in the document]
Comment (taion) [reply commands/new.js]: actually wdyt of dropping `jsnext:main`? it's only supported by old rollup

[the remark this answers is not in the document]
Comment (jquense) [reply commands/new.js]: yes good idea 👍

**Comment by taion** [comment]
yarn dropped a lot of dependencies for some reason... i'm just going to blindly assume that this is fine

**Pull request merged**